### ISSUE TITLE
A detailed Documentation?

### ISSUE THREAD

**Issue by Lowell130** [ponyriders/django-amazon-price-monitor#118]
Hello i am new in Python 3, have you a detailed documentation about installation and configuration of the script? Thank you!

**Comment by dArignac** [comment]
Hi @Lowell130 ,

thanks for checking out our project. First, it isn't a single python script but a collection of services mainly hold together by Django.
Please read our [readme](https://github.com/ponyriders/django-amazon-price-monitor/blob/master/README.rst), within the section "Setup" erverything for setting up is written down. If you encounter any problem, please reply to this issue here. We've written it from our side of view, maybe some things you need are missing. If so, just reply.

Hope that help you. Have a nice day!

Cheers
-Alex

**Comment by Lowell130** [comment]
Hi @dArignac thanks for your reply. I get an error when try migrate "ModuleNotFoundError: No module named 'price_monitor.product_advertising_api'" can you help me about it? Thank you

**Comment by Lowell130** [comment]
Hello @dArignac  thanks for your reply. Currently i'm testing locally the script, any suggestions? Thank you!

**Comment by Lowell130** [comment]
Hello i have a question, but i have to insert single product from admin internface? Or an user can search for producs from frontend form?

**Comment by dArignac** [comment]
You basically don't need the AdminSite. In the frontend you can add a product by filling in the ASIN of it. The price monitor does not search for products, the input is always the product ASIN.

**Comment by Lowell130** [comment]
So  it's a sort of price monitor reminder about amazon products.  Have you experience about use on a Digital Ocean droplet?

**Comment by dArignac** [comment]
Yes, it reminds when a specific price for a specific product is reached. We
wanted to keep things simple, so we did not implement full search of Amazon
products but instead use the ASIN of the appropriate product.

There is no experience on Digital Ocean on our side, we run the price
monitor on dedicated servers.

Am 17.10.2017 3:30 nachm. schrieb "Stew" <notifications@github.com>:

So it's a sort of price monitor reminder about amazon products. Have you
experience about use on a Digital Ocean droplet?

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/ponyriders/django-amazon-price-monitor/issues/118#issuecomment-337231732>,
or mute the thread

**Comment by Lowell130** [comment]
can i see a live demo please, because i had some issues on local testing
thanks

**Comment by dArignac** [comment]
Unfortunately there is no public live demo available.
What were your issues?

**Comment by Lowell130** [comment]
i don't know how start it locally :(

**Comment by dArignac** [comment]
Well, setup is decribed here: https://github.com/ponyriders/django-amazon-price-monitor#id11
There is also a Docker setup here: https://github.com/ponyriders/django-amazon-price-monitor/tree/master/docker - you just need to add the settings for AWS (see here: https://github.com/ponyriders/django-amazon-price-monitor#aws-and-product-advertising-api-credentials) into the compose.env file, as the settings are read from this file (see here: https://github.com/ponyriders/django-amazon-price-monitor/blob/master/docker/web/project/glue/settings.py#L234)
Within the docker folder there is the web folder that contains a Django project that includes the price_monitor - the one docker starts. You can use this as a blueprint project when setting up your own.
One more hint: we only use the docker setup for development, it's not production save in case of security!

**Comment by Lowell130** [comment]
Thank you!
